common/ideas/00_basic_ideas.txt

```diff
@@ -196,159 +104,273 @@ religious_ideas = {
+				}
+			}
+		}
+		modifier = {
+			factor = 0.1
+			NOT = {
+				technology_group = western
+				technology_group = eastern
+				technology_group = muslim
+			}
 		}
 	}
 }
 
-
-offensive_ideas = {
-	category = MIL
+# Might need to come back to this.
+administrative_ideas = {
+	category = ADM
 
 	bonus = {
-		recover_army_morale_speed = 0.05
+		governing_capacity_modifier = 0.33
 	}
 
-	bayonet_leaders = {
-		leader_land_shock = 1
+	organised_mercenaries = {
+		merc_maintenance_modifier = -0.15
+		mercenary_manpower = 0.25
 	}
-	national_conscripts = {
-		global_regiment_recruit_speed = -0.1
+	adaptability = {
+		core_creation = -0.25
 	}
-	superior_firepower = {
-		leader_land_fire = 1
+	improved travel = {
+		local_friendly_movement_speed = 0.2
+		envoy_travel_time = -0.2
 	}
-	glorious_arms = {
-		prestige_from_land = 1.0
+	bookkeeping = {
+		interest = -1
 	}
-	engineer_corps = {
-		siege_ability = 0.20
+	organised_companies = {
+		land_forcelimit_modifier = 0.10
+		hostile_attrition = 1
 	}
-	grand_army = {
-		land_forcelimit_modifier = 0.2
+	centralized_governance = {
+		state_governing_cost = -0.1
 	}
-	napoleonic_warfare = {
-		discipline = 0.05
+	civil_servants = {
+		adm_tech_cost_modifier = -0.1
 	}
 
 	ai_will_do = {
-		factor = 0.94
+		factor = 0.96
+		modifier = {
+			factor = 2
+			has_reform = steppe_horde
+		}
 	}
 }
 
 
-defensive_ideas = {
-	category = MIL
+humanist_ideas = {
+	category = ADM
 
 	bonus = {
-		hostile_attrition = 1
+		development_cost = -0.10
+		yearly_harmony = 0.25  # Used if religion uses harmony (confucianism)
+		brahmins_muslim_loyalty_modifier = 0.1
+		dhimmi_loyalty_modifier = 0.1
 	}
 
-	battlefield_commisions = {
-		army_tradition_from_battle = 1.0
+	tolerance_idea = {
+		religious_unity = 0.25
+		global_unrest = -3
 	}
-	military_drill = {
-		land_morale = 0.15
+	local_traditions = {
+		num_accepted_cultures = 2
+		promote_culture_cost = -0.33
 	}
-	improved_manuever = {
-		leader_land_manuever = 1
+	humanist_tolerance = {
+		tolerance_heretic = 2
+		tolerance_heathen = 2
 	}
-	regimental_system = {
-		land_maintenance_modifier = -0.05
+	indirect_rule = {
+		years_of_nationalism = -10
 	}
-	defensive_mentality = {
-		defensiveness = 0.2
-		fort_maintenance_modifier = -0.1
+	humanist_education = {
+		technology_cost = -0.05
+		idea_cost = -0.15
 	}
-	supply_trains = {
-		reinforce_speed = 0.33
+	benevolence = {
+		improve_relation_modifier = 0.33
+		diplomatic_upkeep = 1
 	}
-	improved_foraging = {
-		land_attrition = -0.25
+	developed_people = {
+		advisor_cost = -0.15
 	}
 
 	ai_will_do = {
-		factor = 0.77
+		factor = 0.9
+		modifier = {
+			factor = 0
+			has_idea_group = religious_ideas
+		}
+	}
+}
+
+
+# DIP Ideas
+
+spy_ideas = {
+	category = DIP
+
+	bonus = {
+		rebel_support_efficiency = 0.50
+	}
+
+	efficient_spies = {
+		spy_offence = 0.66
+		fabricate_claims_cost = -0.25
+	}
+
+	agent_training = {
+		diplomats = 1			
+		advisor_cost = -0.1
+	}
+
+	remote_surveillance = {
```

Comment:
We need to remember to add a tooltip to explain the intel monarch point mechanic

Reply:
I was thinking we could have that in the Idea description.